lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2633,6 +2641,29 @@ static function ( $pseudo_selector ) use ( $selector ) {
 				}
 				unset( $declarations[ $index ] );
 			}
+
+			if ( $is_root_selector && (
+					'background-size' === $declaration['name'] && 'cover' === $declaration['value'] ||
```

Comment:
Just wondering, would we also want the min-height rule to be output if we use `contain` as the background size? I.e. should we always output the min height if any background image is set, a little like how we're outputting if any `background` shorthand property is in use? 🤔

Reply:
> Just wondering, would we also want the min-height rule to be output if we use contain as the background size?

Good question. I don't know, but I'll test it out.

>  I.e. should we always output the min height if any background image is set, a little like how we're outputting if any background shorthand property is in use?

We could I suppose, so long as it behaves as expected with other possible background property values. I'll also play around with it.